cmake/Tidy.cmake

```diff
@@ -19,8 +19,16 @@ else()
     message(FATAL_ERROR "clang-tidy version ${CMAKE_MATCH_1} is too low")
 endif()
 
+# Find Python and run-clang-tidy script
+find_package(Python 3 REQUIRED)
```

Comment:
Why do we need a python script on Windows?
Do we need this also on other OS?
Just wondering if the requirement for Python as dependency applies to all OS

Reply:
run-clang-tidy is a python script, it’s just that on macos and Linux they can be run as executable scripts whereas on windows you have to invoke python. I wanted all OSes to use the same code so now we invoke python on all OSes.